.hermetic_builds/add_manual_build_dependencies.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "pipenv==2025.0.3" >> requirements-build.in
```

Comment:
@patchkez can you remove it from here as well please? Probably drop the whole file and its use in makefile target.